hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/config/YamlJetMemberConfigLocator.java

```diff
@@ -30,7 +30,7 @@
     private static final String HAZELCAST_ENTERPRISE_MEMBER_DEFAULT_YAML = "hazelcast-jet-enterprise-member-default.yaml";
 
     public YamlJetMemberConfigLocator() {
-        super(true);
+        super(false);
```

Comment:
I guess this means `dont fail if the suffix is not expected`, why ?

Reply:
It shouldn't fail to be able to fallback to other methods like loading from classpath etc.